app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -3045,8 +3103,9 @@ public function getFirstScopeFromConfig($path, $value)
         $collection = Mage::getResourceModel('core/config_data_collection')
             ->addFieldToFilter('path', array('eq' => $path))
             ->addFieldToFilter('value', array('eq' => $value));
+
         if ($collection->getSize()) {
-            $configEntry = $collection->getFirstItem();
+            $configEntry = $collection->setPageSize(1, 1)->getLastItem();
```

Comment:
Move setPageSize filter to the collection definition to keep everything together.
setPageSize function only takes 1 parameter.

Reply:
Solved.